lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -187,4 +187,26 @@ export class PublicClientApplication extends ClientApplication implements IPubli
             }
         }
     }
+
+    /**
+     * Sets the account to use as the active account. If no account is passed to the acquireToken APIs, then MSAL will use this active account.
+     * @param account 
+     */
+    setActiveAccount(account: AccountInfo | null): void {
+        if (this.broker) {
+            this.broker.setActiveAccount(account);
+        }
+        super.setActiveAccount(account);
+    }
+
+    /**
+     * Gets the currently active account
+     */
+    getActiveAccount(asBroker?: boolean): AccountInfo | null {
+        if (asBroker) {
```

Comment:
What are the use cases for setting this to `true` or `false`?

Reply:
If you are attempting to get the broker's active account, pass `true`, otherwise pass `false`. Basically, if you are in a brokered scenario or not. I had to do this since the broker's active account is different from the PCA's active account, and there isn't a way for these two apps to share them today.

Reply:
I think those should always be the same account.

Reply:
It was brought up in email that storing the active account in browser storage would be valuable, sounds like it would help here as well.

Reply:
Resolved offline